application/server/lsp/message_types.go

```diff
@@ -783,3 +785,18 @@ type IacIssueData struct {
 	Path          []string `json:"path"`
 	References    []string `json:"references,omitempty"`
 }
+
+type IacIssuePathToken struct {
+	Value any
+}
+
+func (c *IacIssuePathToken) String() string {
```

Comment:
@ofekatr Could you please move the conversion logic to the IaC scanner, when parsing IaC results and converting them to `snyk.Issue` representation? 

You should not be touching anything in `application/` or `domain/` at all.

Reply:
Fixed in https://github.com/snyk/snyk-ls/pull/269/commits/b866fde0c95531333e06c107927fe63999a87c71

Reply:
Sorry, you've probably misunderstood what I meant. Could you have the logic in [this function](https://github.com/snyk/snyk-ls/blob/6d2a941c53e0effd4d5cd1c9e7f43143c506000f/infrastructure/iac/iac.go#L376)? It is part of IaC scanner and should belong there.

The changes outside of scanner shouldn't happen. Happy to have a quick call if you need little assistance 👍

Reply:
Fixed in 8a2796a1b0f68622658e470ccf6b839ef2665088